packages/interface/src/components/JoinButton.tsx

```diff
@@ -249,26 +264,22 @@ export const JoinButton = (): JSX.Element => {
           variant={isRegistered === undefined || isLoading ? "disabled" : "primary"}
           onClick={handleZupassVerify}
         >
-          Prove you can join with Zupass
+          Prove you voted with Zupass
         </Button>
       </div>
     );
   }
 
-  if (isEligibleToVote && !isRegistered) {
+  if (isEligibleToVote && poapLink !== "") {
```

Comment:
would this show if a user is recognized as signed up?

Reply:
yes, only when is signed up, as you will only get a poapLink if you signed up